src/stable/lib/state.ts

```diff
@@ -189,17 +189,23 @@ globalThis._azleDispatch = (action: Action): void => {
     }
 
     if (action.type === 'DELETE_AZLE_REJECT_CALLBACK') {
-        delete globalThis._azleRejectCallbacks[action.payload];
+        if (globalThis._azleRejectCallbacks) {
```

Comment:
Please use explicit conditional checks, for example `globalThis._azleRejectCallbacks !== undefined`. Do this for all checks that you have added.